docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -1150,6 +1152,96 @@ The file to import the slashing protection database from. The file must be forma
 
 Predefined network configuration. Accepts a predefined network name. The default is `TESTNET`.
 
+### `watermark-repair`
+
+Updates the slashing protection low watermark for validators.
```

Comment:
nit: Maybe we should add a small explanation of what the low watermark is

Reply:
@jframe Can you provide or point me to a good definition of it?

Reply:
Web3Signer will not sign any block that has a slot lower than the block watermark. Similarly, Websigner will not sign any attestations that have a source epoch less than the watermark source epoch or a target epoch less than the watermark target epoch. There are low watermarks for both the block slot and attestation source and targets and these are kept for each validator.